packages/playwright-core/src/server/bidi/bidiInput.ts

```diff
@@ -33,7 +33,7 @@ export class RawKeyboardImpl implements input.RawKeyboard {
 
   async keydown(modifiers: Set<types.KeyboardModifier>, code: string, keyCode: number, keyCodeWithoutLocation: number, key: string, location: number, autoRepeat: boolean, text: string | undefined): Promise<void> {
     const actions: bidi.Input.KeySourceAction[] = [];
-    actions.push({ type: 'keyDown', value: getBidiKeyValue(key) });
+    actions.push({ type: 'keyDown', value: getBidiKeyValue(code) });
```

Comment:
Shall `getBidiKeyValue = (key: string)` be as well get updated to reflect that `code` has to be passed in and not `key`? That explains why I didn't find the problem. Thanks for fixing it!

Reply:
Yes, renaming makes sense, feel free to send a PR.

Reply:
(the function has been copied from puppeteer)